alibi_detect/cd/classifier.py

```diff
@@ -94,12 +94,7 @@ def __init__(
         """
         super().__init__()
 
-        backend = backend.lower()
-        if backend == 'tensorflow' and not has_tensorflow or backend == 'pytorch' and not has_pytorch:
-            raise ImportError(f'{backend} not installed. Cannot initialize and run the '
-                              f'ClassifierDrift detector with {backend} backend.')
-        elif backend not in ['tensorflow', 'pytorch']:
-            raise NotImplementedError(f'{backend} not implemented. Use tensorflow or pytorch instead.')
+        _validate_backend(backend)
```

Comment:
This would need to be done for all the detectors. Running:

`grep -r "{backend} not implemented. Use tensorflow or pytorch instead" alibi_detect/*` will show where else it needs to be added.